tests/system/fim/common_tasks/verify_alerts.yml

```diff
@@ -3,28 +3,28 @@
     find:
       paths: "{{ agents_results_manager_path }}/{{ event }}"
     register: find_result
-    when: 
+    when:
       - inventory_hostname in groups['masters']
 
   - name: "Verify alert json log | {{ event }}"
     script: >
-      verify_alerts_json.py -i {{ item.path }} -e {{ event }} 
-      -o {{ missing_alerts_json_path }}
+      verify_alerts_json.py -i {{ item.path }} -e {{ event }}
+      -o {{ missing_alerts_json_path }} {{ alert_verification_extra_arguments }}
```

Comment:
``{{ elastic_verification_extra_arguments }}`` is also an alert verification, please consider changing it to `{{ json_verification_extra_arguments }} as it's related to `alerts.json` verification

Reply:
But what if those scripts accept different parameters in the future?

Reply:
The `{{ elastic_verification_extra_arguments }}` will accept multiple parameters and will be declared using `set_fact`. E.G:

```yml
set_fact:
   elastic_verification_extra_arguments: "-s 60 -w True"
```